apisix/plugins/proxy-rewrite.lua

```diff
@@ -178,6 +183,28 @@ function _M.rewrite(conf, ctx)
         ctx.var.upstream_uri = upstream_uri
     end
 
+    -- enum    = {"GET", "POST","PUT","DELETE"}
+    -- change Http method
+    if conf.method ~=nil then
```

Comment:
Style: missing empty space before `nil`.

Reply:
There's an 'else' statement.

Reply:
Hi, @54corbin 
`tokers` mean: `if conf.method ~=nil then` needs to be changed to `f conf.method ~= nil then`.